doc/dev/tests-advanced.md

```diff
@@ -162,25 +168,7 @@ class ExampleStorageTestCase(AzureTestCase):
             table = client.create_table(valid_table_name)
 ```
 
-The first test is the same as above, the second test tries to create a table that already exists and asserts that the correct type of error is raised in response. These tests use cached preparers unlike the previous example.
+The first test is the same as above. The second test tries to create a table that already exists and asserts that the correct type of error is raised in response. These tests use cached preparers, unlike the previous example.
 
 Notes:
-1. The cached preparers here will first look to see if an existing resource group or storage account exists with the given parameters, in this case the `name_prefix`. For more information on what parameters differentiate a new resource group or storage account look for the `self.set_cache()` method in the preparer source code [here](https://github.com/Azure/azure-sdk-for-python/blob/main/tools/azure-sdk-tools/devtools_testutils/storage_testcase.py#L49). The advantage to using a cached preparer is the time saver to re-using the same resource instead of creating a new resource for each test. However, this can increase the possibility that you have to be more exact about cleaning up the entities created in between test runs.
-
-## mgmt_settings_real file
-
-A `mgmt_settings_real.py` can be used in place of a `.env` file by copying `sdk/tools/azure-sdk-tools/devtools_testutils/mgmt_settings_fake.py` to `sdk/tools/azure-sdk-tools/devtools_testutils/mgmt_settings_real.py` and providing real credentials to it. The following changes need to be made to the `mgmt_settings_real.py` file:
```

Comment:
Do we want to talk about auth at all here? I know we don't want mgmt_settings_real to be mentioned, but the alternative is to strongly encourage folks to use the testresources option right? Because mgmt_settings_real only came into play for the older `preparers`, right?

Reply:
I'll take a second look at this and see if there's documentation we should put here, good point!